packages/typeorm-decorator-ismoney/__tests__/index.ts

```diff
@@ -0,0 +1,69 @@
+import { validate } from 'class-validator';
+import { IsMoney, IsMoneyValidator } from '../src';
+
+describe('IsMoney validation', () => {
+  describe('Validator', () => {
+    it('accepts valid money object', () => {
+      const validator = new IsMoneyValidator();
+      expect(validator.validate({ amount: 100, currency: 'EUR' })).toBeTrue();
+    });
+
+    it('rejects money object with missing amount', () => {
+      const validator = new IsMoneyValidator();
+      expect(validator.validate({ currency: 'EUR' })).toBeFalse();
+    });
+
+    it('rejects money object with negative amount', () => {
+      const validator = new IsMoneyValidator();
+      expect(validator.validate({ amount: -12, currency: 'EUR' })).toBeFalse();
```

Comment:
I think money can be negative

Reply:
it might you need to have additional option for negative check

Reply:
IsMoney validation on gateway prohibited negative amounts. I took it from there.